[Asset] Document remote JSON manifest (#13255)
symfony/symfony-docs: components/asset.rst

    echo $package->getUrl('css/app.css');
    // result: build/css/app.b916426ea1d10021f3f17ce8031f93c2.css

> If the JSON file not on the filesystem, but it is accessible using HTTP, use the

Reviewer: ```suggestion
If your JSON file is not on your local filesystem but is accessible over HTTP,
you can use the ...
```

what about this?
